### DISCUSSION TITLE
Jetbrains Rider Requires Admin Mode To Work

### DISCUSSION THREAD

**Discussion by RacerDelux** [Eptagone/Vite.AspNetCore#137] [General]
To start, on Mac running Rider, no issues.

On windows though, when I try to run the `ViteNET.MVC` project, it fails to run unless I start Rider in admin mode. Is this a configuration issue or a bug?
To note, works in Visual Studio on Windows just fine.

To note: On Mac, I did have to manually create the directory it was trying to insert certs into. The Vite dev server failed to start until I did that.

`info: Vite.AspNetCore.ViteDevServerLauncher[21]
      Starting the Vite development server...
info: Vite.AspNetCore.ViteDevServerLauncher[22]
      Vite development server started with process ID 25548.
warn: Vite.AspNetCore.ViteDevServerLauncher[27]
      The Vite development server did not start within 5 seconds
`

**Comment by Eptagone** [comment]
Hi, Maybe it's a configuration problem. I don't have that problem with Rider.

**Reply by RacerDelux** [reply]
This is the demo code I downloaded from the repo, nothing has been changed

**Reply by Eptagone** [reply]
I mean some configuration with Rider installation or something.
